lib/rules.js

```diff
@@ -97,6 +97,35 @@ LiveRules.prototype.getAll = function (cb) {
   });
 };
 
+/**
+* Get rules matching GNIP IDs
+* @param ids List of strings
+* @param cb Function callback
+*/
+LiveRules.prototype.getByIds = function (ids, cb) {
+  var self = this;
+  var json = {
+    rule_ids: ids
+  };
+  request.post({
+    url: self._api + '?_method=get',
+    json: json,
+    headers: { 'Authorization': self._auth }
+  }, function (err, response, body) {
+    if (err) cb(err);
+    else if (response.statusCode >= 200 && response.statusCode < 300) {
+      try {
+        cb(null, body.rules);
```

Comment:
How exactly can this statement fail? Accessing an undefined property won't raise an error.
Maybe you forgot to call `self.parser.parse(body).rules` like the method above?

Reply:
You're right.  I was trying to follow the model from above as closely as possible and I overdid it :).  I originally thought gnip would return a string like it does when it returns all rules and then I would have had to parse; but then I realized it actually returns regular json, obviously - and forgot to clean up the try/catch.  I don't think the try/catch is doing any harm there other than adding unnecessary boilerplate.  But I'm happy to clean up and resubmit!!

Reply:
Nice catch @demian85! 

@aniham I think it would be nice to clean it up. We can then merge the changes to master and have it included in a future release. Thanks!